src/components/accessories/laboratory/examForm/ExamForm.tsx

```diff
@@ -349,6 +417,31 @@ const ExamForm: FC<ExamProps> = ({
                 disabled={isLoading}
               />
             </div>
+            <div className="patientExamForm__item">
+              <AutocompleteField
+                fieldName="material"
+                fieldValue={formik.values.material}
+                label={t("lab.material")}
+                isValid={isValid("material")}
+                errorText={getErrorText("material")}
+                onBlur={onBlurCallback("material")}
+                isLoading={materialsLoading}
+                options={materialsOptionsSelector(materialsList)}
+                disabled={isLoading}
+              />
+            </div>
+            <div className="patientExamForm__item">
+              <AutocompleteField
+                fieldName="status"
+                fieldValue={formik.values.status}
+                label={t("lab.status")}
+                isValid={isValid("status")}
+                errorText={getErrorText("status")}
+                onBlur={onBlurCallback("status")}
+                options={statusOptions}
+                disabled={isLoading}
+              />
```

Comment:
We should turn this components into a simple select list.

Reply:
I didn't edit this field. Also, there could be more twenty exams, so an autocomplete field is suitable.